service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageDocumentsController.java

```diff
@@ -49,6 +49,8 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
 
         caseDetails.getData().putAll(manageDocumentService.initialiseManageDocumentEvent(caseData));
 
+        caseDetails.getData().remove("furtherEvidenceDocumentsTEMP");
```

Comment:
We should make it a task to monitor if this field is present in any of the cases in a week or so and if it isn't we can remove this